.github/workflows/remoteconfig.yml

```diff
@@ -0,0 +1,44 @@
+name: remoteconfig
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    paths:
+    - 'FirebaseRemoteConfig**'
+    - '.github/workflows/remoteconfig.yml'
+    - 'Gemfile'
+  schedule:
+    # Run every day at 11pm (PST) - cron uses UTC times
+    - cron:  '0 7 * * *'
+
+jobs:
+
+
+  quickstart:
+    env:
+      plist_secret: ${{ secrets.GHASecretsGPGPassphrase1 }}
+      signin_secret: ${{ secrets.GHASecretsGPGPassphrase1 }}
+    runs-on: macOS-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: technote-space/load-config-action@v1
+      with:
+        CONFIG_FILENAME: workflows/config.yml
+    - name: Set xcode version
+      run: sudo xcode-select -switch /Applications/Xcode_"${{env.XCODE_VERSION}}".app
+    - name: Setup quickstart
+      run: scripts/setup_quickstart.sh config
+    - name: Install Secret GoogleService-Info.plist
+      run: scripts/decrypt_gha_secret.sh scripts/gha-encrypted/qs-auth.plist.gpg \
+          quickstart-ios/config/GoogleService-Info.plist "$plist_secret"
+    - name: Install Secret FIREGSignInInfo.h
+      run: scripts/decrypt_gha_secret.sh scripts/gha-encrypted/FIREGSignInInfo.h.gpg \
+          quickstart-ios/TestUtils/FIREGSignInInfo.h "$signin_secret"
+    - name: Test quickstart
+      run: |
+        cd ./quickstart-ios/config/
+        xcodebuild -workspace *.xcworkspace -list
+        xcodebuild test -scheme ConfigExample -workspace ConfigExample.xcworkspace -destination "platform=${{env.PLATFORM}},name=${{env.DEVICE_NAME}},OS=${{env.OS_VERSION}}"
```

Comment:
We have a script to do this for us already without the config:

https://github.com/firebase/firebase-ios-sdk/blob/24e23931b43c7791b456d92eede67ebd88ae525d/.github/workflows/storage.yml#L58-L59

(replacing `Storage` with `Config` or whatever the sample is named).